circuitpy_examples_fall24/10-21-fun_things_in_Winona.md

```diff
@@ -5,4 +5,4 @@
 | Fishing for panfish | food + something pulling back on the fishing line | [the lake or the river](https://www.dnr.state.mn.us/gofishing/where-fish.html) | license+[pole](https://www.winona.edu/outdoor-education-recreation/) | Continuous in Winona | Nathan|
 | Cross Country Skiing | getting outside in the winter | [St. Mary's University](https://saintmaryssports.com/sports/2016/2/18/NSC_0218161559.aspx) | SMU rents skis | January-February | Nathan and [lots of elemetary school kids](https://www.myxc.org/clubs/winona) |
 | Scream at the Lake | screaming | [the lake](https://www.google.com/maps/place/Lake+Lodge+Recreation+Center,+Winona+Park+%26+Rec/@44.0383193,-91.6446129,2168m/data=!3m1!1e3!4m14!1m7!3m6!1s0x87f96c52a035e83d:0xcdf4b961e7152176!2sLake+Winona!8m2!3d44.0379541!4d-91.6384287!16s%2Fg%2F1hhhp7ltt!3m5!1s0x87f96dad14ad1153:0xebbf9b1730ab4ed9!8m2!3d44.040866!4d-91.64458!16s%2Fg%2F11lghgbp6n?entry=ttu&g_ep=EgoyMDI0MTAxNi4wIKXMDSoASAFQAw%3D%3D) | Free | All | Josh M |
-
+| Open Skate at Bud King | Ice Skating | [Bud Kind Ice Arena] 670 E Front St, Winona, MN 55987 | $3 admission + $2 skate rentals | Select times year round | Carly |
```

Comment:
I don't think this link works.  Maybe this would be better?

[Bud King Ice Arena](https://www.cityofwinona.com/facilities/facility/details/budkingicearena-31)

Also, maybe there will be skating at Lake Park this year?